meta-intel-edison-bsp/recipes-bsp/u-boot/u-boot-common_2021.10.inc

```diff
@@ -27,6 +27,10 @@ SRC_URI += "file://btrfs.cfg"
 SRC_URI += "file://0001-REVERTME-usb-dwc3-gadget-skip-endpoints-ep-18-in-out.patch"
 SRC_URI += "file://0001-x86-tangier-Enable-support-for-SD-SDIO-family-in-the.patch"
 SRC_URI += "file://0002-x86-edison-Don-t-take-SD-card-detect-pin-into-consid.patch"
+SRC_URI += "file://netconsole.cfg"
```

Comment:
For some reason this patch is preventing `./ﬂashall.sh --recover` (and also manual attempts using `xfstk`) to work. I verified by reverting.

So in the final PR I will drop this, possibly until at a later time if we can figure out a way to have `netconsole` simultaneous with `xfstk` flashing.